metafacture-flux/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
   api project(':metafacture-framework')
   implementation project(':metafacture-commons')
   implementation project(':metafacture-io')
+  implementation project(':metafacture-plumbing')
```

Comment:
Why was this dependency introduced?

Reply:
This was needed to test the `Tee` command.

Reply:
I see, but then it should only be a `testRuntime` dependency, right?